lib/build/amodro-trace/write/stubs.js

```diff
@@ -12,7 +13,9 @@ module.exports = function stubs(options) {
   options = options || {};
 
   return function(context, moduleName, filePath, contents) {
-    if (options.stubModules && options.stubModules.indexOf(moduleName) !== -1) {
+    if (options.stubModules
```

Comment:
I've asked the guy behind Amodrotrace if he'd be open to us starting pull requests for features that we need